src/OrbitBase/Profiling.cpp

```diff
@@ -38,4 +42,17 @@ uint64_t EstimateClockResolution() {
   }
   return minimum_resolution_found;
 }
+
+uint64_t EstimateAndLogClockResolution() {
```

Comment:
I don't really understand why this was moved here. The commit message says "so that it is accessible on Windows" but where is that the case? This was just to log clock resolution at the beginning of OrbitService so its original place made more sense to me.

Reply:
This will be useful to log the resolution on the Client, for introspection. Also, depending on how the Windows service code is laid out, it's handy to have this function available from multiple translation units.